imod/tests/test_mf6/test_mf6_dsp.py

```diff
@@ -61,6 +90,15 @@ def test_dispersion_options():
 
     assert actual == expected
 
+def test_dispersion_erronous_options():
+    message = textwrap.dedent(
```

Comment:
I wouldn't test on the message. In most pytest.raises tests this is not done, and it will cause a test to fail if the message is edited for some reason.

Reply:
maybe we should go for numbered error messages then. Then we can check on the error number rather than on the precise string. That's more an idea for the future and would not be part of this PR.